src/features/movie/components/imageContainer/imageContainer.module.scss

```diff
@@ -0,0 +1,79 @@
+@import "../../../../assets/styles/variables/variables.scss";
+@import "../../../../assets/styles/mixins/mixins.scss";
```

Comment:
Same comment for imports as above.

Reply:
I don't think it works with scss, but I'm trying to find out solution.

Reply:
I found a solution ->  @import "~assets/styles/variables/variables.scss";

Reply:
I found a solution ->  @import "~assets/styles/variables/variables.scss";